### PR TITLE
Update fsspec 2022.10.0

### PR DESCRIPTION
**Pull request by skshetry**

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -27,7 +27,7 @@ install_requires=
     tqdm>=4.63.1,<5
     shortuuid>=0.5.0
     funcy>=1.14
-    fsspec>=2022.5.0
+    fsspec>=2022.10.0
```
Comment (skshetry) [inline setup.cfg:30]: This version bump may cause us some pain.

**Comment by efiop** [comment]
Need to check that all filesystems work with this (e.g. that s3fs, gcsfs, adlfs, etc don't pin older versions).

**Comment by skshetry** [comment]
@efiop, that can be done downstream I think. `gcsfs` always pins IIRC.

**Comment by skshetry** [comment]
The only one that is not compatible is gcsfs, I have sent a PR to bump the version. Other than that, other filesystems should be fine. We can wait for that package in downstream dvc as well. Merging…

**Pull request merged**